adapters/intertech/intertech.go

```diff
@@ -0,0 +1,231 @@
+package intertech
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v3/adapters"
+	"github.com/prebid/prebid-server/v3/config"
+	"github.com/prebid/prebid-server/v3/errortypes"
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+	"github.com/prebid/prebid-server/v3/util/jsonutil"
+)
+
+const (
+	pageIDMacro = "{{page_id}}"
+	impIDMacro  = "{{imp_id}}"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+type ExtImpIntertech struct {
```

Comment:
Why are you defining a local `ExtImpIntertech` rather than using the one in `openrtb_ext`? And why are they different?

The expectation is that `openrtb_ext` documents all the bidder specific `ext` fields, so there is a collected reference for them all. So it should be an unusual circumstance that requires you to define a separate, local version of it.